src/modules/launcher/Wox.Plugin/Result.cs

```diff
@@ -112,17 +111,24 @@ public Result(IList<int> titleHighlightData = null, IList<int> subTitleHighlight
         /// </summary>
         public IList<int> GetTitleHighlightData()
```

Comment:
Where do we use the public methods for the get / set. Can't just the in-line get / private set work?" For the var?